scripts/common/host-packages.sh

```diff
@@ -114,9 +130,11 @@ function _dpkg_install_host_packages() {
         return 0
     fi
 
-    DEBIAN_FRONTEND=noninteractive dpkg --install --force-depends-version --force-confold "${fullpath}"/*.deb
+    DEBIAN_FRONTEND=noninteractive dpkg --install --force-depends-version --force-confold --auto-deconfigure "${fullpath}"/*.deb
```

Comment:
For future consideration, should we start using `apt-get` instead to leverage its dependency management?

Reply:
We cannot use apt-get because we are providing the packages, Therefore, it might fails when should not because tried to looking for a host packaged that is required and will be installed as part of the kURL install. 

Also, we need to keep in mind that the apt-get will only be able to fix the dependencies and get the new requirements when those are found available in the package manager index, which is updated from the repositories configured in the specific machine. Therefore, the user host might have configured a different repository from where we got the package, and it might fail in a use case that would still be able to work out. 

It seems that calling dpkg and the apt-get install -f is kind of the same as calling apt-get install ./deb directly. (Look [here](https://unix.stackexchange.com/questions/159094/how-to-install-a-deb-file-by-dpkg-i-or-by-apt) might help to better understand the thought)

Reply:
What I meant was that `apt-get install ./deb` will go ahead and resolve dependencies in addition to installing the local debian package. This way, you would not need to run `sudo apt-get check status`. Either approach should be fine by me.

Reply:
However, yes described above we cannot use `apt-get install ./deb` we must use dpkg to do the installation.